rpm/valgrind.spec

```diff
@@ -27,6 +27,13 @@ malloc/new/free/delete are intercepted. As a result, Valgrind can
 detect a lot of problems that are otherwise very hard to
 find/diagnose.
 
+%package extratools
```

Comment:
Dependency is added automatically like it was previously done for the main package.

Don't think perl on package name is a good choice, callgrind user shouldn't need to know which programming language was used for the implementation. It's not valgrind-c either :)